data/sql/updates/db_world/2025_05_18_00.sql

```diff
@@ -0,0 +1,6 @@
+-- DB update 2025_05_15_00 -> 2025_05_18_00
+
+-- Reduce Aku'Mai Servant damage. Auto attack = 36-45 w 1.5k armor
+UPDATE 'creature_template' SET 'DamageModifier' = 1.7 WHERE 'entry' = 4978 AND 'type' = 4;
+ -- Reduce Deviate Moccasin damage. Auto attack = 39-44 w 1.1k armor
+UPDATE 'creature_template' SET 'DamageModifier' = 1.8 WHERE 'entry' = 5762 AND 'family' = 35;
```

Comment:
```suggestion
UPDATE `creature_template` SET `DamageModifier` = 1.7 WHERE `entry` = 4978 AND `type` = 4;
 -- Reduce Deviate Moccasin damage. Auto attack = 39-44 w 1.1k armor
UPDATE `creature_template` SET `DamageModifier` = 1.8 WHERE `entry` = 5762 AND `family` = 35;
```
For table and columns, use backticks (`). For entering text values, use apostrophes ('), such as:
```sql
UPDATE `creature_template` SET `name` = 'Example Text' WHERE `name` = 'Also Example Text';
```